src/plugins/kibana/public/management/sections/indices/upload/index.js

```diff
@@ -0,0 +1,15 @@
+import management from 'ui/management';
+import routes from 'ui/routes';
+import template from 'plugins/kibana/management/sections/indices/upload/index.html';
+import './directives/upload_wizard';
+
+routes.when('/management/data/upload/', {
+  template: template
+});
+
+management.getSection('data').register('upload', {
+  display: 'Upload CSV',
+  order: 10,
+  path: 'data/upload/'
```

Comment:
yeah, my initial thought was that this wizard might handle additional data types like JSON files in the future, which is why I called it upload. But the more I think about it, we would probably have a totally different link for different data types, so I'll switch it to data/csv instead